app/controllers/sign_up/recovery_codes_controller.rb

```diff
@@ -36,7 +36,7 @@ def next_step
       elsif current_user.password_reset_profile.present?
         reactivate_profile_path
       else
-        after_sign_in_path_for(current_user)
+        profile_path
```

Comment:
I'm pretty sure that in this scenario, `after_sign_in_path` will always be profile_path, so might as well go there directly but I will revert this since it's not really related to this PR.